parlai/core/distributed_utils.py

```diff
@@ -31,9 +31,9 @@ def validate_params(opt):
 
     Raises exceptions if anything is wrong, otherwise returns None.
     """
-    if torch.version.__version__.startswith('0.'):
+    if torch.__version__ < "1.1.0":
```

Comment:
i think this check and the one in `pytorch_data_teacher.py` are fine for now

Reply:
Do you mean I roll back to the previous check(```if torch.version.__version__.startswith('0.'):```) 
or keep the one I made (```if torch.__version__ < "1.1.0":```) ?

Reply:
Use your new one, but can we unify them so they're all one call?

utils.check_torch_version()

so we don't have to upgrade in many places